packages/core/contracts/financial-templates/funding-rate-store/implementation/FundingRateStore.sol

```diff
@@ -10,13 +10,15 @@ import "../../../oracle/interfaces/OracleInterface.sol";
 import "../../../oracle/interfaces/FinderInterface.sol";
 import "../../../oracle/implementation/Constants.sol";
 
+import "../../perpetual-multiparty/PerpetualInterface.sol";
 import "../interfaces/FundingRateStoreInterface.sol";
 import "../../../common/implementation/Testable.sol";
 import "../../../common/implementation/Lockable.sol";
+import "../../../common/implementation/Withdrawable.sol";
 import "../../../common/implementation/FixedPoint.sol";
 
 
-contract FundingRateStore is FundingRateStoreInterface, Testable, Lockable {
+contract FundingRateStore is FundingRateStoreInterface, Withdrawable, Testable, Lockable {
```

Comment:
Why make this withdrawable?

Reply:
No reason at the current implementation, but as discussed in [this comment](https://github.com/UMAprotocol/protocol/pull/2157#discussion_r519061259) we might want to allow the Perpetual to withdraw some fees in an emergency shutdown event. The question though is who should withdraw, and how much.